packages/account/src/Components/self-exclusion/self-exclusion-inputs.tsx

```diff
@@ -436,13 +424,12 @@ const MaximumDepositLimitInputs = () => {
 };
 
 const SelfExclusionInputs = () => {
-    const { is_appstore } = React.useContext(PlatformContext);
     const { footer_ref, goToConfirm, is_app_settings } = React.useContext(SelfExclusionContext);
     const { dirty, isSubmitting, isValid, values }: TFormikContext = useFormikContext<TFormikContext>();
     const versions: Record<string, { condition: boolean; components: Array<React.FunctionComponent> }> = {
         // Global settings for account for DWallet.
         dwallet: {
```

Comment:
@fasihali-deriv help to remove all the instances of Dwallet from the code!

It is referring to the old version of Wallets and since it is related to `is_appstore`, this PR should be a right place to remove all those!

Reply:
Removed DWallet. 
Thanks @matin-deriv for highlighting.